### DISCUSSION TITLE
Ran the commands as suggested. It allowed Borg to run manually but the next scheduled backup failed.

### DISCUSSION THREAD

**Discussion by madman2012** [nextcloud/all-in-one#4544] [Questions] [answered]
### Steps to reproduce
1.
2.
3.

#### Other valuable info

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot reproduce this unfortunately. Did it fail with the same error again? Can you post the mastercontainer logs after it failed?

**Reply by madman2012** [reply]
Turns out I disabled the auto backup reenabled it and changed runtime and that worked. It's been working for weeks now without a problem.
